src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -91,6 +117,11 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
       }
     }
     else {
+
+      if ($this->configuration['handler_settings']['field_mode'] != 'admin') {
```

Comment:
Similar to above, i think I have the logic working in https://github.com/amitaibu/og/pull/21 so we will want to either use that, or this.